docs/UsersGuide/source/AQM.rst

```diff
@@ -0,0 +1,164 @@
+
+.. code-block:: console
+
+   git checkout ff6f103
+
+This hash will then check out the following hashes (as of 03/08/2023) of the external components, which are specified in ``ufs-srweather-app/Externals.cfg``:
+
+.. _ExternalsAQM:
+
+.. table:: Externals for Online-CMAQ
+
+   +--------------------+--------------+
+   | Component          | Hash         |
+   +====================+==============+
+   | UFS_UTILS          | ca9bed8      |
+   +--------------------+--------------+
+   | ufs-weather-model	| e051e0e      |
+   +--------------------+--------------+
+   | UPP                | 2b2c84a      |
+   +--------------------+--------------+
+   | NEXUS              | 3842818      |
+   +--------------------+--------------+
+   | AQM-utils          | e078c70      |
+   +--------------------+--------------+
+
+Users may replace the hashes above with different ones if they prefer. For example, users can comment out the hash line and uncomment the branch line with a new ``repo_url`` address to use a different branch for development. In the example below, the ``ufs-weather-model`` repository URL has been changed to check out code from a user's personal fork rather than from the authoritative UFS repository. 
+
+.. code-block:: console
+
+   repo_url = https://github.com/chan-hoo/ufs-weather-model
+   branch = feature/for_example
+   #hash = ff6f103
+
+Checkout Externals
+---------------------
+
+Along with the components detailed in :numref:`Chapter %s <Components>`, the AQM version of the SRW App pulls in the externals listed in :numref:`Table %s <ExternalsAQM>`. Users must run the ``checkout_externals`` script to collect (or "check out") the individual components of the SRW App (AQM version) from their respective GitHub repositories. 
+
+.. code-block:: console
+
+   ./manage_externals/checkout_externals
+
+Build the SRW App with AQM
+-----------------------------
+
+On Hera and WCOSS2, users can build the SRW App AQM binaries with the following command:
+
+.. code-block:: console
+
+   ./devbuild.sh -p=<machine> -a=ATMAQ
+
+where ``<machine>`` is ``hera``, or ``wcoss2``. The ``-a`` argument indicates the configuration/version of the application to build. 
+
+Building the SRW App with AQM on other machines, including other `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ platforms, is not currently guaranteed to work, and users may have to make adjustments to the modulefiles for their system. 
+
+Load the ``regional_workflow`` Environment
+--------------------------------------------
+
+Load the python environment for the workflow:
+
+.. code-block:: console
+
+   # On WCOSS2 (do not run on other systems):
+   source ../versions/run.ver.wcoss2
+   # On all systems (including WCOSS2):
+   module use /path/to/ufs-srweather-app/modulefiles
+   module load wflow_<machine>
+   conda activate regional_workflow
```

Comment:
@gspetro-NOAA, this `conda activate` command is not necessary on WCOSS2.

Reply:
@chan-hoo Is any further action required on WCOSS2? On other systems, there is a console message saying to run `condo activate`. Does WCOSS2 have a different message, no message? Just want to clarify in the docs.